webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -240,23 +241,55 @@ export default class CreateIssueModal extends PureComponent {
             </React.Fragment>
         );
 
-        if (jiraIssueMetadata && jiraIssueMetadata.error) {
-            component = (
+        let issueError = null;
+        let component;
+
+        // if no getMetaDataError, show fields and allow user to input
+        // fields. An error at this point is from a server-side create
+        // issue submission and should be displayed (in addition to fields)
+        // to the user after clicking the create button.
+        if (error) {
+            issueError = (
                 <div style={style.modal}>
-                    {jiraIssueMetadata.error}
+                    <React.Fragment>
```

Comment:
Also here, not sure if we need it...?
Maybe these were here from sometime before, but may as well clean them up.